tests/unit/llm/test_litellm.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest.mock import AsyncMock, MagicMock, Mock, patch
+
+import nest_asyncio
+import pytest
+from distilabel.llm.litellm import LiteLLM
+
+
+@patch("litellm.acompletion")
+class TestLiteLLM:
+    def test_litellm_llm(self, _: MagicMock) -> None:
+        llm = LiteLLM(model="gpt-4", api_key="api.key")  # type: ignore
+        assert isinstance(llm, LiteLLM)
+        assert llm.model_name == "gpt-4"
```

Comment:
Could you add more tests with the different providers that `LiteLLM` offers? i.e. one function per provider? or at least the most common ones

```suggestion
    def test_litellm_llm(self, _: MagicMock) -> None:
        llm = LiteLLM(model="gpt-4", api_key="api.key")  # type: ignore
        assert isinstance(llm, LiteLLM)
        assert llm.model_name == "gpt-4"
```